Cargo.toml

```diff
@@ -1,9 +1,9 @@
-[profile.release]
+# [profile.release]
 lto = "thin"
```

Comment:
?

Reply:
Similarly puzzled how that happened. Reverting that of course.